detox/src/devices/drivers/android/genycloud/GenyCloudDeviceAllocator.js

```diff
@@ -47,7 +51,7 @@ class GenyCloudDeviceAllocator extends AndroidDeviceAllocator {
     }
 
     const options = {
-      backoff: 'none',
+      backoff: 'none', // TODO apply reverse-linear polling
```

Comment:
What is everse-linear polling? backoff strategy?

Reply:
Exactly